internal/node/node.bzl

```diff
@@ -131,6 +132,12 @@ _NODEJS_EXECUTABLE_ATTRS = {
         Enable this to get stack traces that point to original sources, e.g. if the program was written
         in TypeScript.""",
         default = True),
+    "configuration_env_vars": attr.bool(
+        doc = """Pass configuration environment variables to the resulting binary
+        Enable this to get configuration environment variables (taken from ctx.var), which also
+        includes anything specified via the --define flag, which can lead to different outputs
+        produced by this rule.""",
+        default = True),
```

Comment:
hmm, I agree that this was a hasty addition, we shouldn't pollute the environment of every program with the entire list.
I think instead of a boolean, it should be a string_list.